pkg/diagnostics/network/objects.go

```diff
@@ -13,7 +13,7 @@ import (
 )
 
 const (
-	diagnosticsImage           = "openshift/diagnostics-deployer"
+	diagnosticsImage           = "openshift/origin"
```

Comment:
Can this be updated to use the image format string so that we don't have to patch OCP?